docs/UserGuide.md

```diff
@@ -98,13 +98,15 @@ Examples:
 
 Adds a transaction to the transaction book.
 
-Format: `addTxn p/PHONE_NUMBER amt/AMOUNT desc/TEST [date/DATE]`
+Format: `addTxn [p/PHONE_NUMBER] [amt/AMOUNT] [desc/TEST] [date/DATE] [cat/CATEGORY]...`
```

Comment:
The `[ ]` indicates that the fields are optional, in this case only date and cat should be optional I think
<img width="777" alt="Screenshot 2024-10-29 at 10 46 20 AM" src="https://github.com/user-attachments/assets/58195c32-3561-48ca-bb0c-81724e04cd05">
![telegram-cloud-photo-size-5-6086956776244231810-y](https://github.com/user-attachments/assets/19688125-1e14-41f3-9efb-c3f5d85db043)